### ISSUE TITLE
[Modal] with detachable=false is not working with 2.3

### ISSUE THREAD

**Issue by ZyoxNet** [Semantic-Org/Semantic-UI#6206]
Hi,

with 2.3 the undetachable modal is not working correctly anymore. 

See https://jsfiddle.net/ca0rovs3/221/

**Comment by foreveRush** [comment]
Hi, I also have css issue with Controlled modal... After updating to version 2.3

**Comment by manasilele** [comment]
` $('.ui.tiny.modal').modal({detachable: false, centered: true }).modal('show');`

If `detachable:false` is removed, then the modal is displayed in the center. Otherwise it aligns itself on the top right. 
Has anyone found a fix yet?

**Comment by hluz** [comment]
bump!

**Comment by Jielde** [comment]
does not work with patch 2.3.2 :(

**Comment by TastesLikeTurkey** [comment]
Ugh... This bug just brought development of one of my pages to a grinding halt.

**Comment by manasilele** [comment]
@tastesliketurkey
If your page does not postback and you have all the data pulled using ajax, you can just avoid having detachable false

**Comment by TastesLikeTurkey** [comment]
@manasilele Yeah, I need to post back, and the modal contains data I need. 🙁 I got around it by just manually rendering the dimmer and modal where I want them, but it still would have been nice to stick with the default implementation.

**Comment by manasilele** [comment]
@TastesLikeTurkey  does it align in the center though?

**Comment by TastesLikeTurkey** [comment]
Do you mean now that I'm rendering it manually? If so, then yes. Here's what I'm doing:

```
// Inside my form. I'm using Razor Pages, so this is not rendered if the modal shouldn't be shown.
<div class="ui dimmer modals active" style="position:fixed;">
    <div class="ui active modal">
        // Modal content
    </div>
</div>
```

**Comment by AlexanderZnaydyuk** [comment]
I spent a whole day trying to fix my modals and bugs caused because of 2.3 version of semantic.

The best solutions right now make a rollback to 2.2

npm install semantic-ui@2.2.14
gulp build

The modals are fixed, the labels are fixed.

**Comment by melkalakhi** [comment]
Try this :
`$('#your-modal-id').modal({ useFlex: false}).modal("show");
$('.ui.dimmer.modals').appendTo('#aspnetForm');
$('.ui.dimmer.modals').css('overflow', 'scroll');
$('#your-modal-id').css('position', 'relative');`
